content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
Yeah this doesn't work correctly, as the `HtmlWebpackPlugin` dynamically adds the assets, but they're also hard-coded in the index.html file.

We can do two things:

1. Drop this altogether, just not use a template
2. Use a template, but make it dynamic.

We removed the 2nd option because the guide was getting quite long and people can read up on how to use the `HtmlWebpackPlugin` templates in the respective docs.

I would recommend removing it for now, is that okay?